src/main/java/com/riwi/localstorage/riwi_local_storage/api/dto/request/create/MembershipRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.riwi.localstorage.riwi_local_storage.api.dto.request.create;
+
+import org.hibernate.validator.constraints.Length;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@Data
```

Comment:
please do not use the @Data change it to the setter and getter and the builder in the Dtos is not necessary.